packages/fxa-auth-server/test/local/routes/validators.js

```diff
@@ -289,25 +289,28 @@ describe('lib/routes/validators:', () => {
     });
 
     it('accepts a well-formed http:// URL', () => {
-      const res = v.validate('http://example.com/path');
-      assert.ok(!res.error);
+      const schema = Joi.string().uri();
+      const res = schema.validate('http://example.com/path');
```

Comment:
this works but the describe has a line const v = validators.redirectTo(); so we somehow have to chain onto that instead of making a new const.